docs/subnets/reference-cli-commands.md

```diff
@@ -0,0 +1,517 @@
+    --genesis string      file path of genesis to use
+-h, --help                help for create
+    --latest              use latest VM version, takes precedence over --vm-version
+    --spacesvm            use the SpacesVM as the base template
+    --vm string           file path of custom vm to use
+    --vm-version string   version of vm template to use
+```
+
+### delete
+
+The subnet delete command deletes an existing Subnet configuration.
+
+**Usage:**
+
+`avalanche subnet delete [flags]`
+
+**Flags:**
+
+```
+-h, --help help for delete
+```
+
+### deploy
+
+The subnet deploy command deploys your subnet configuration locally, to
+Fuji Testnet, or to Mainnet.
+
+At the end of the call, the command will print the RPC URL you can use
+to interact with the subnet.
+
+Subnets may only be deployed once. Subsequent calls of deploy to the
+same network (local, Fuji, Mainnet) are not allowed. If you'd like to
+redeploy a subnet locally for testing, you must first call [avalanche
+network clean](#clean) to reset all deployed chain state. Subsequent local
+deploys will redeploy the chain with fresh state. The same subnet can
+be deployed to multiple networks, so you can take your locally tested
+subnet and deploy it on Fuji or Mainnet.
+
+**Usage:**
+
+`avalanche subnet deploy [subnetName] [flags]`
+
+**Flags:**
+
+```
+    --avalanchego-version string   use this version of avalanchego (ex: v1.17.12) (default "latest")
+    --control-keys strings         addresses that may make subnet changes
+-f, --fuji testnet                 deploy to fuji (alias to testnet
+-h, --help                         help for deploy
+-k, --key string                   select the key to use [fuji deploy only]
+-g, --ledger                       use ledger instead of key (always true on mainnet, defaults to false on fuji)
+-l, --local                        deploy to a local network
+-m, --mainnet                      deploy to mainnet
+    --output-tx-path string        file path of the blockchain creation tx
+-s, --same-control-key             use creation key as control key
+    --subnet-auth-keys strings     control keys that will be used to authenticate chain creation
+-t, --testnet fuji                 deploy to testnet (alias to fuji)
+    --threshold uint32             required number of control key signatures to make subnet changes
+```
+
+### describe
+
+The subnet describe command prints the details of a subnet configuration
+to the console. By default, the command will print a summary of the
+configuration. By providing the --genesis flag, the command will instead
+print out the raw genesis file.
+
+**Usage:**
+
+`avalanche subnet describe [subnetName] [flags]`
+
+**Flags:**
+
+```
+-g, --genesis   Print the genesis to the console directly instead of the summary
+-h, --help      help for describe
+```
+
+### export
+
+The subnet export command prints the details of an existing subnet deploy.
```

Comment:
prints? rather, exports?

Reply:
Good catch, it actually writes it to a file.